packages/nestjs-common/tsconfig.json

```diff
@@ -1,33 +1,12 @@
 {
-  "extends": "@voiceflow/tsconfig",
+  "extends": "@voiceflow/tsconfig/monorepo.json",
   "compilerOptions": {
-    // General
-    "target": "es2021",
-    "module": "commonjs",
-    "lib": ["es2021"],
-    "experimentalDecorators": true,
-    "baseUrl": "./",
+    "baseUrl": ".",
+    "rootDir": "src",
+    "outDir": "build",
     "paths": {
       "@nestjs-common/*": ["src/*"]
-    },
-
-    // Strictness
-    "strict": true,
```

Comment:
moved a lot of these up to our `@voiceflow/tsconfig/root.json` file which all other configs inherit from
removed flags we weren't using